code-reviews/faq.md

```diff
@@ -0,0 +1,37 @@
+# FAQ
+
+This is a list of questions / frequently occurring issues when working with code reviews and answers how you can possibly tackle them.
+
+## We experience very large PRs, how can we fix this?
+
+Make sure you size the work items smaller. They should be able to be completed on its own. The team is instructed to commit early, before the full product backlog item / user story is complete, but rather when an individual item is done. If the work would result in an incomplete feature, make sure it can be turned off, until the full feature was delivered.
+More information can be found in [Pull Requests - Size Guidance](./pull-requests.md#size-guidance).
+
+## We experience slow code reviews, causing delays in delivering features
+
+### Possible actions you can take:
+
+- Add a rule for PR turnaround time to your work agreement.
+- Setup a slot after the standup to go through pending PRs and assign the ones that are inactive
```

Comment:
This and the following line seem to be the outliers on using periods at the end of bullet items or not with both of these not having them but others all having them.  Would recommend adding here.

Reply:
done.